test/onnx/test_pytorch_onnx_shape_inference.py

```diff
@@ -1,72 +1,69 @@
 import unittest
 import torch
+import numpy as np
 
-import copy
 
-import test_pytorch_onnx_onnxruntime
-from test_pytorch_onnx_onnxruntime import TestONNXRuntime
-from torch.onnx import utils, OperatorExportTypes, TrainingMode
-from torch.onnx.utils import _validate_dynamic_axes
-from torch.onnx.symbolic_helper import (_set_opset_version, _set_operator_export_type,
-                                        _set_onnx_shape_inference, _set_training_mode,
-                                        _is_tensor_list, _is_tensor, _is_none)
+def expect_tensor(scalar_type, shape=None, dynamic_shape=None):
+    def verify(actual_type):
```

Comment:
Seems that there is only code in `expect_tensor` is `verify`.
Can we just use the code in `verify` instead of having this function?

Reply:
`expect_tensor` returns a function object. `expect_tensor` serves as api for devs to pass in expected shape/dtype info. The returned function object takes actual type as argument, and applies test assertion.